__tests__/components/Link.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Link } from '../../src/components';
+
+describe('Link', () => {
+  it('renders a default Link', () => {
+    const tree = renderer.create(<Link description={'description'} url={'url'} />).toJSON();
```

Comment:
here ist a `.toJSON()` and in other tests not. what is the difference in usage here and there?

Reply:
thanks, was a mistake , corrected in 9d3b4be5b92af2f9b3b9669be38ae70ad29230e1

Reply:
what was the mistake? do you know the difference in using `toJSON()` or not here?

Reply:
https://jestjs.io/docs/25.x/cli#--json 
Prints the test results in JSON and this is not what we need ?

Reply:
`toJSON()` in code and `--json` for cli has nothing to do with each other. have you read about jest and `toJSON()`?

Reply:
In [jest docs](https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest) it's not really explained. 
And  [stackoverflow](https://stackoverflow.com/questions/61284054/using-tojson-or-not-with-snapshot-testing-with-jest) has this explanation; **"I would suggest calling toJSON just to follow "explicit is better than implicit" principle and the fact that the inner behavior is subject to change."** 
So I actually think it's a better practice and more specific to use toJSON() in snapshot testing although I'm not 100% sure I understand what does it do.

Reply:
ef1041b4785f639c1b81b3ffb5cd5d8338149e2e